public/app/features/org/OrgPreferences.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import Tooltip from '../../core/components/Tooltip/Tooltip';
+import SimplePicker from '../../core/components/Picker/SimplePicker';
+import { DashboardAcl, OrganizationPreferences } from 'app/types';
+import {
+  setOrganizationHomeDashboard,
+  setOrganizationTheme,
+  setOrganizationTimezone,
+  updateOrganizationPreferences,
+} from './state/actions';
+
+export interface Props {
+  preferences: OrganizationPreferences;
+  starredDashboards: DashboardAcl[];
```

Comment:
this has be the wrong type,

Reply:
Yeah, forgot to change it. Used Acl as placeholder. Should we go with DashboardSearchHit?